plugins/modules/cloud/misc/terraform.py

```diff
@@ -177,24 +177,31 @@
 import os
 import json
 import tempfile
+from distutils.version import LooseVersion
 from ansible.module_utils.six.moves import shlex_quote
 
 from ansible.module_utils.basic import AnsibleModule
 
-DESTROY_ARGS = ('destroy', '-no-color', '-force')
-APPLY_ARGS = ('apply', '-no-color', '-input=false', '-auto-approve=true')
 module = None
 
 
-def preflight_validation(bin_path, project_path, variables_args=None, plan_file=None):
+def get_version(bin_path):
+    extract_version = module.run_command([bin_path, 'version', '-json'], use_unsafe_shell=True)
```

Comment:
```suggestion
    extract_version = module.run_command([bin_path, 'version', '-json'])
```
Or why is `use_unsafe_shell` needed?

Reply:
I don't no exeactly why may be a historical approach